Addition of purpose of UG, amendment of introduction and quick start (#142)
AY2021S1-CS2113T-F14-3/tp: docs/UserGuide.md

## 2. Quick Start
The following steps will guide you through the process of running **Nav@NUS**.

1. Ensure that you have Java `11` or above installed in your computer. If you do not have it installed,
follow the guide [here](https://docs.oracle.com/en/java/javase/11/install/installation-jdk-microsoft-windows-platforms.html#GUID-A7E27B90-A28D-4237-9383-A58B416071CA).
2. Download the latest `Nav@NUS.jar` from [here](https://github.com/AY2021S1-CS2113T-F14-3/tp/releases).
3. Open command prompt on your computer.
Orientate yourself to the command line interface. As seen in the figure below, the red arrow points to
where you have to type in commands.<br>
![Windows CLI](UG_Figures/windowsCLI.png)

3. Copy the jar file to the folder you want to use as the _home folder_ for Nav@NUS.jar bus application. In the example
shown in the figure, the home folder is found in the address path of "C:\Users...\CS2113T Empty folder".<br>
![Windows CLI](UG_Figures/windowsPath.png)
4. In the command prompt, type `cd` and the directory of the _home folder_. Press <kbd>Enter</kbd> to continue.
5. Run the .jar file in the command prompt as follows by typing `java -jar duke.jar` and press <kbd>Enter</kbd>.

> 6. Your screen should show the start screen of Nav@NUS.<br>

Author: ```suggestion
6. Your screen should show the start screen of Nav@NUS as seen in the figure below.<br>
```

commute in NUS to be as effortless as possible. Nav@NUS uses a CLI to facilitate quick typing and retrieval of 
information that you require.

Nav@NUS consists of 3 main features:

* **Route**: Searches for bus routes from your location to your intended destination.
* **Dine**: Seeks dining options for you to explore culinary world of NUS.
* **Fav**: Saves your commands for you to have a personalised user experience catered to your needs.

Skip the tight squeeze near information boards and use Nav@NUS today!

## 2. Quick Start
The following steps will guide you through the process of running **Nav@NUS**.

1. Ensure that you have Java `11` or above installed in your computer. If you do not have it installed,
follow the guide [here](https://docs.oracle.com/en/java/javase/11/install/installation-jdk-microsoft-windows-platforms.html#GUID-A7E27B90-A28D-4237-9383-A58B416071CA).
2. Download the latest `Nav@NUS.jar` from [here](https://github.com/AY2021S1-CS2113T-F14-3/tp/releases).
3. Open command prompt on your computer.
Orientate yourself to the command line interface. As seen in the figure below, the red arrow points to

> where you have to type in commands.

Reviewer: ```suggestion
where you have to type in commands.<br>
```

![Windows CLI](UG_Figures/windowsCLI.png)

> Figure 1.1 Windows CLI

Reviewer: Maybe remove the figure numbers for now, since we removed it from other places also, if the figure is right below the content
```suggestion
```

3. Copy the jar file to the folder you want to use as the _home folder_ for Nav@NUS.jar bus application. In the example

> shown in Figure 1.3, the home folder is found in the address path of "C:\Users...\CS2113T Empty folder".

Reviewer: ```suggestion
shown in the figure, the home folder is found in the address path of "C:\Users...\CS2113T Empty folder".<br>
```

![Windows CLI](UG_Figures/windowsPath.png)

> Figure 1.3 Path of home folder

Reviewer: ```suggestion
```

4. In the command prompt, type `cd` and the directory of the _home folder_. Press <kbd>Enter</kbd> to continue.
5. Run the .jar file in the command prompt as follows by typing `java -jar duke.jar` and press <kbd>Enter</kbd>.

> 6. Your screen should show the start screen of Nav@NUS.

Reviewer: ```suggestion
6. Your screen should show the start screen of Nav@NUS as seen in the following figure.<br>
```

![Start Screen](UG_Figures/Nav@NUSstartScreen.png)

> Figure 1.4 Start screen of Nav@NUS

Reviewer: ```suggestion
```
